dev-manuals/test-data-management/README.md

```diff
@@ -0,0 +1,48 @@
+# Test data management
+
+## Introduction
+
+Since we have a lot of separate databases (at least one for each backend service), we need to have an easy way to initialize them with test data.
```

Comment:
```suggestion
Since we have a lot of separate databases (at least one for each backend), we need to have an easy way to initialize them with test data.
```